api/src/main/java/io/opencensus/common/Duration.java

```diff
@@ -78,6 +79,20 @@ public static Duration fromMillis(long millis) {
     return Duration.create(seconds, nanos);
   }
 
+  /**
+   * Converts a {@link Duration} to milliseconds.
+   *
+   * <p>Note that there could be overflow or loss of precision by making this conversion. See {@link
+   * TimeUnit#convert(long, TimeUnit)} for details.
```

Comment:
I think it would be better to describe the overflow behavior without mentioning TimeUnit, since TimeUnit is an implementation detail, and its documentation is more general.  Also, it looks like TimeUnit.toMillis returns Long.MIN_VALUE or Long.MAX_VALUE instead of overflowing.  I don't think there is any loss of precision, besides the unavoidable conversion from nanoseconds to milliseconds.

Reply:
Done - I copied the description from TimeUnit.toMillis here. I'm not sure if we should mention that converting nanoseconds to milliseconds will cause precision loss though (seems users should have known that).

Reply:
> I'm not sure if we should mention that converting nanoseconds to milliseconds will cause precision loss though (seems users should have known that).

I agree that we don't need to mention the precision loss.